crates/cheatnet/tests/cheatcodes/declare.rs

```diff
@@ -1,8 +1,15 @@
 use crate::common::state::create_cheatnet_state;
 use crate::common::{get_contracts, state::create_cached_state};
-use cairo_felt::Felt252;
 use cheatnet::cheatcodes::{CheatcodeError, EnhancedHintError};
 use conversions::StarknetConversions;
+use starknet_api::core::ClassHash;
+use starknet_api::hash::StarkFelt;
+use starknet_api::stark_felt;
+
+static HELLO_STARKNET_CLASS_HASH: &str =
+    "0x298f80e468953d1e65314b6bc63347c7a3fe454a89c2b15387dd52ee281d103";
+static CONSTRUCTOR_SIMPLE_CLASS_HASH: &str =
+    "0x02dbeae7583f3dd4af0bc2da4d58611433165fec7e31245bfa2f1378fbff6a4c";
```

Comment:
Calculated using starknet py I guess?

Reply:
with `sn_cast`, mentioned it in PR description